blueprint/index.md

```diff
@@ -0,0 +1,311 @@
+                        },
+                        {
+                            "name": "tAnswered",
+                            "value": 11179,
+                            "emitDate": "2021-04-01T12:28:04.569Z"
+                        }
+                    ],
+                    "provider": "Webchat Provider",
+                    "remote": "John Doe",
+                    "requestedRoutings": [
+                        "Standard"
+                    ],
+                    "usedRouting": "Standard",
+                    "selectedAgentId": "50fcd33c-f432-4726-abc9-979f3ca01555"
+                }
+            ]
+        }
+    ]
+}
+```
+
+## Important notes
+
+The database design used in this blueprint will not store the full schema from the [POST 
+  /api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) response, some of the data was omitted to simplify the design. The full schema can be seen at the `200 - successful operation` section of the API documentation.  
+
+The following steps outline how to get driver script running to show this design in action. It is intended as a possible starting point for developers looking to map conversation responses or other non-flat JSON data to SQL. This logic is not limited to typescript, people looking to access the SQL queries can find them in [db.ts](https://github.com/GenesysCloudBlueprints/conversation-model-to-sql-blueprint/blob/main/src/db.ts).  
+
+## Implementation steps
+
+* [Clone the repository containing the project files](#clone-the-repository-containing-the-project-files)
+* [Create a Client Credentials OAuth Grant for Genesys Cloud](#create-a-client-credentials-oauth-grant-for-genesys-cloud)
+* [Start the docker container](#start-the-docker-container)
+* [Install the node dependencies](#install-the-node-dependencies)
+* [Create the database tables](#create-the-database-tables)
+* [Insert some sample data](#insert-some-sample-data)
+* [Fetch the data and compare](#fetch-the-data-and-compare)
+* [Invoke the analytics conversations details API](#invoke-the-analytics-conversations-details-api)
+
+### Clone the repository containing the project files
+
+1. Clone the [Conversation model to SQL Blueprint](https://github.com/GenesysCloudBlueprints/conversation-model-to-sql-blueprint "Opens the conversation-model-to-sql-blueprint repository in GitHub") repository.
+
+### Create a Client Credentials OAuth Grant for Genesys Cloud
+
+1. If you haven't already got Client Credentials with the required scopes, log in to your Genesys Cloud organization and create a new OAuth client that uses the Client Credentials Grant type with the required scopes. For more information, see [Create an OAuth client](https://help.mypurecloud.com/articles/create-an-oauth-client/ "Opens the Create an OAuth client article") in the Genesys Cloud Resource Center.
```

Comment:
> haven't already got

reword: "don't already have". Feel free to hang on to "have got" for when we offer localization in Irish/UK English. The British may have invented English, but Americans perfected it. YEEHAW! ;) (I kid, but I believe our writing standards are generally defined in US English)

Reply:
Yeah that's a bit of hiberno English sneaking through :D

Reply:
Dont use my english as a good example of American english :)